.vscode/settings.json

```diff
@@ -4,12 +4,15 @@
         "rpc_call",
         "rpc_test",
         "import",
-        "pathfinder"
+        "pathfinder",
+        "get_nonce"
     ],
     "cSpell.words": [
         "Deoxys",
         "rstest",
         "serde",
-        "starknet"
+        "STARKGATE",
```

Comment:
Please git ignore .vscode

Reply:
On it!

Reply:
Removed `.vscode` in commit ef7a5ac864d56ec7819cdf02f61ec85c822d10b5